### DISCUSSION TITLE
Custom Font?

### DISCUSSION THREAD

**Discussion by JackCuyler** [Eugeny/tabby#4686] [Q&A]
Is there a way to use a custom font for the terminal?

**Comment by Eugeny** [comment]
Yes - in Settings -> Appearance

**Reply by JackCuyler** [reply]
Sorry, I was unclear. Settings -> Appearance has a list of fonts to choose from. Is there a way to add a custom font to that list?

**Reply by Eugeny** [reply]
No problem. It lists all installed monospace fonts on the system. If for some reason a font doesn't show up (e.g. it's not declared as monospace), you still can type in its name manually and it will work.

**Reply by stephaneeybert** [reply]
So I have downloaded my custom font from https://github.com/ryanoasis/nerd-fonts/releases/download/v2.1.0/LiberationMono.zip

How can I add it to the list and select it ?

**Reply by Eugeny** [reply]
Just install it in your OS - Tabby simply lists all installed fonts, same as every other app.

**Reply by stibinator** [reply]
I have Fira mono installed (with the fancy ligatures) and it's not showing up. Neither is my other favourite coding font jetBrains mono. Or Sourcecode Pro. They're in %appdata%\Microsoft\windows\fonts\ rather than c:\windows\fonts

**Reply by stibinator** [reply]
Yeah, there's something very messed up about the way it's handling fonts. It seems to only show the non-monospaced fonts, I can't choose any of my monospaced fonts. And the result is illegible

Here are my monospaced fonts:
![Screenshot 2022-06-23 154349](https://user-images.githubusercontent.com/422938/175224029-27991989-4463-491f-bc2a-5c43d18e4965.jpg)

and here's what I get if I try to search for them:
![Screenshot 2022-06-23 154429](https://user-images.githubusercontent.com/422938/175224094-4b96864b-22fb-4568-b8c4-640a40efeb1f.jpg)

And here's the garbled porridge that displays when I choose one of the fonts that it lets me choose (source sans). It looks like it's forcing the font into mono spacing.
![Screenshot 2022-06-23 154643](https://user-images.githubusercontent.com/422938/175224309-a016879f-d287-444e-adf5-00e077f3fec1.jpg)

**Reply by sirianni** [reply]
Old thread, but +1.  I'm having the same issue on MacOS.  Other than `Menlo`, none of the other monospace fonts show up.
